src/home/sorting/filter-issues.ts

```diff
@@ -0,0 +1,33 @@
+import { taskManager } from "../home";
+
+export function filterIssues(textBox: HTMLInputElement, issuesContainer: HTMLDivElement) {
```

Comment:
please rename this file and functions to: filter-issues-by-search.ts and filterIssuesBySearch respectively.